cmd/kubelet/app/server.go

```diff
@@ -264,6 +264,7 @@ func Run(s *options.KubeletServer, kcfg *KubeletConfig) error {
 	if kcfg == nil {
 		cfg, err := UnsecuredKubeletConfig(s)
 		if err != nil {
+			glog.Errorf("Unable to generate a kubelet config: %v", err)
```

Comment:
LGTM in that case. One more thing to consider is to add a simple wrapper to avoid having to log explicitly whenever this method is updated.